lib/workos/profile.rb

```diff
@@ -48,7 +51,7 @@ def to_json(*)
 
     private
 
-    # rubocop:disable Metrics/AbcSize
+    # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
```

Comment:
@willmanduffy @marktran How do we feel about this rubocop configuration change since it's a pretty large hash?

Reply:
These are both good! This cop is a bit aggressive to be honest!

Reply:
sgtm! 👍